integration-cli/docker_cli_attach_test.go

```diff
@@ -94,13 +92,6 @@ func (s *DockerSuite) TestAttachTtyWithoutStdin(c *check.C) {
 	id := strings.TrimSpace(out)
 	c.Assert(waitRun(id), check.IsNil)
 
-	defer func() {
```

Comment:
@MHBauer the container will be deleted in teardown so I think there's no need to kill it in a deferred func.